content/docs/reference/reproducibility.md

```diff
@@ -0,0 +1,41 @@
++++
+title="Reproducible Builds"
+weight=9
++++
+
+## Summary
+The Cloud Native Buildpacks project aims to create "Reproducible Builds" of container images. For image creation commands (`create-builder`, `package-buildpackag`, `build`) `pack` aims to create in a reproducible fashion. "Reproducible" is hard to define but we'll do so by example:
+
+---
+Running `pack build` produce a container image with the same image ID (*local* case)
```

Comment:
produce > produces ?

Reply:
Do you mean to say running pack **multiple times** with the same inputs should produce the same output? I think the "multiple" part might be worth emphasizing in the title.